pkg/runner/runner_test.go

```diff
@@ -2,21 +2,31 @@ package runner
 
 import (
 	"io/ioutil"
+	"os"
+	"path/filepath"
 	"testing"
 
 	"github.com/kubeshop/testkube/pkg/api/v1/testkube"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestRunFiles(t *testing.T) {
+	// setup
+	tempDir := os.TempDir()
+	os.Setenv("RUNNER_DATADIR", tempDir)
+
+	k6_script, err := ioutil.ReadFile("k6-test-script.js")
```

Comment:
k6Script - usually in go variables don't have underscores

Reply:
Renamed the variable.